app/admin/block.rb

```diff
@@ -0,0 +1,28 @@
+ActiveAdmin.register Block do
+  index do
+    selectable_column
+    column :id
+    column :user
+    column :title
+    actions
+  end
+
+  show do
+    attributes_table do
+      row :id
+      row :user
+      row :title
+    end
+    active_admin_comments
+  end
+
+  form do |f|
```

Comment:
Unused block argument - `f`. You can omit the argument if you don't care about it.